Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -35,6 +35,13 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
         this._fontPropertiesMap = new Map;
         this._fontVariationsMap = new Map;
         this._fontFeaturesMap = new Map;
+
+        this._fontStyles = null;
+        this._fontVariationRowsMap = new Map;
+        this._basicPropertyRowsMap = new Map;
+        this._basicPropertyNames = ["font-size", "font-style", "font-weight", "font-stretch"];
+
+        this._abortController = new AbortController;
```

Comment:
I'm going to let you explain what an AbortController is in our meeting tomorrow 😅